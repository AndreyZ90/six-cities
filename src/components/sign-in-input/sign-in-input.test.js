import React from 'react';
import TestRenderer from 'react-test-renderer';

import SignInInput from '@/components/sign-in-input/sign-in-input';

const type = `password`;
const label = `Password`;
const value = `test@gmail.com`;

describe(`SignInInput component snapshot`, () => {
  test(`Should correctly render SignInInput component`, () => {
    const tree = TestRenderer.create(
        <SignInInput
          type={type}
          label={label}
          value={value}
          onDataChange={() => {}}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
