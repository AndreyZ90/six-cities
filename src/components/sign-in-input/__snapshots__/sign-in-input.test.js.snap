// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInInput component snapshot Should correctly render SignInInput component 1`] = `
<div
  className="login__input-wrapper form__input-wrapper"
>
  <label
    className="visually-hidden"
  >
    Password
  </label>
  <input
    className="login__input form__input"
    name="password"
    onChange={[Function]}
    placeholder="Password"
    required={true}
    type="password"
    value="test@gmail.com"
  />
</div>
`;
